### DISCUSSION TITLE
How to pull the PR Comments given by reviewer in any sprint/particular duration?

### DISCUSSION THREAD

**Discussion by SurekhaShankar** [community/community#36403] [Pull Requests]
I'm looking for an option to extract the PR Comments given by reviewers for a release/sprint/any particular duration.
The intention is to extract the review comments provided, analyze for root cause and act to reduce repeatable comments.
Can anyone help to extract the review comments?
Note: the queries are not responding the accurate answers. 
Thanks.

**Comment by ossanna16** [comment]
Hey there @SurekhaShankar and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by spenserblack** [comment]
You can get PR comments from the REST api. For example, using [`gh`](https://github.com/cli/cli), it would be `gh api repos/OWNER/REPO/pulls/ID/comments`.

You can read more about it here: https://docs.github.com/en/rest/pulls/comments
